packages/nhost_ferry_adapter/lib/postgres_types.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:built_value/serializer.dart';
+
```

Comment:
This entire file needs to either be replaced by something more robust, or pulled into its own lib.